src/index.js

```diff
@@ -2,8 +2,10 @@ import configValidator from './configValidator'
 
 import checkConfig from './checkConfig'
 import messageFormatter from './messageFormatter'
+import uncoveredPaths from './uncoveredPaths'
 
 module.exports = configValidator
```

Comment:
Only in this file because I couldn't get the required to work properly for people using CJS